crates/fluvio-sc/src/services/public_api/remote/list.rs

```diff
@@ -0,0 +1,37 @@
+use fluvio_auth::AuthContext;
+use fluvio_sc_schema::{
+    core::MetadataItem,
+    objects::{ListFilters, ListResponse, Metadata},
+    remote::RemoteSpec,
+};
+use anyhow::Result;
+use tracing::{debug, info, trace};
+
+use crate::services::auth::AuthServiceContext;
+
+pub async fn handle_list_remote<AC: AuthContext, C: MetadataItem>(
+    _filters: ListFilters,
+    auth_ctx: &AuthServiceContext<AC, C>,
+) -> Result<ListResponse<RemoteSpec>> {
+    info!("remote-cluster list");
+    let remote_list: Vec<Metadata<RemoteSpec>> = auth_ctx
+        .global_ctx
+        .remote()
+        .store()
+        .read()
+        .await
+        .values()
+        .map(|item| {
```

Comment:
Isn't the `item` owned value here, so you can just destruct it and move values instead of cloning?

Reply:
I changed to: 

```
.map(|item| item.inner().clone().into())
```

I think that I can't reduce more than one clone here. But I am not sure. 🤔